website/views/shared/avatar/appearance.jade

```diff
@@ -15,6 +15,9 @@ mixin avatar(opts)
     each klass, item in visualBuffs
       span(ng-if='profile.stats.buffs.#{item}', class='#{klass}')
 
+    // Show flower if using Flower visual buff
+    span(ng-if='profile.stats.buffs.shinySeed', class='hair_flower_{{profile.preferences.hair.flower}}')
```

Comment:
@crookedneighbor @SabreCat This is what they look like:
![screenshot 2016-06-04 at 6 06 26 pm](https://cloud.githubusercontent.com/assets/12386291/15802647/2c6dd9aa-2a7f-11e6-81f9-eb46758923bd.png)
![screenshot 2016-06-04 at 6 06 11 pm](https://cloud.githubusercontent.com/assets/12386291/15802648/333bc878-2a7f-11e6-8197-c0a1799f53f1.png)
![screenshot 2016-06-04 at 6 05 56 pm](https://cloud.githubusercontent.com/assets/12386291/15802649/3a0e77d6-2a7f-11e6-9cf4-3efc4cfa2080.png)
They don't look very good.